client/src/components/SessionForm.jsx

```diff
@@ -0,0 +1,221 @@
+import React, { useState } from 'react';
+import { Modal, Form, Input, TextArea, Radio } from 'semantic-ui-react';
+
+import 'semantic-ui-css/semantic.min.css';
+import { addSession } from '../utils/apiWrapper';
+import '../css/SessionForm.scss';
+
+function SessionForm(props) {
+  const { button } = props;
+  const [open, setOpen] = useState(false);
+  const [isLater, setIsLater] = useState(false);
+  const [courseCode, setCourseCode] = useState('');
+  const [courseNumber, setCourseNumber] = useState();
+  const [courseSuffix, setCourseSuffix] = useState('');
+  const [location, setLocation] = useState('');
+  const [attendees, setAttendees] = useState('');
+  const [notes, setNotes] = useState('');
+  const [date, setDate] = useState();
+  const [startTime, setStartTime] = useState();
+  const [endTime, setEndTime] = useState();
+  const [endTimeDefined, setEndTimeDefined] = useState(false);
+
+  const processFormAndSubmit = async () => {
+    const creatorID = 'jaskdlfioawh';
+    const course = courseCode + courseNumber + courseSuffix;
+    const attendeeArray = attendees.split(',');
```

Comment:
A next step (no need to add this to this PR): find a more user-friendly way to list attendees on the form (like a dropdown), since it's easy to make a mistake when separating attendees by commas